src/CSSTransition.js

```diff
@@ -150,44 +141,62 @@ class CSSTransition extends React.Component {
     const isStringClassNames = typeof classNames === 'string';
     const prefix = isStringClassNames && classNames ? classNames + '-' : '';
```

Comment:
```suggestion
    const prefix = isStringClassNames && classNames ? `${classNames}-` : '';
```

just making some un-related cleanups (: